recommendations/unknown-summary.md

```diff
@@ -1,2 +1,7 @@
-* We could not provide a recommendation for this issue.
-* Something else might be running in the system causing interference.
+- There is a performance issue but Clinic could not accurately categorize it.
+- Clinic may inadvertently be picking up system noise.
+    - Disable any CPU- or Memory-intensive applications.
+    - Use the `--on-port` flag to immediately begin load testing, this is known to reduce the chances of interference.
```

Comment:
if we are testing an http application, it's simper to use `--autocannon`

Reply:
added to doc